[2025-01-30T04:37:59.158+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-30T04:37:59.195+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etapa2_dag.copia manual__2025-01-30T03:32:35.404173+00:00 [queued]>
[2025-01-30T04:37:59.211+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etapa2_dag.copia manual__2025-01-30T03:32:35.404173+00:00 [queued]>
[2025-01-30T04:37:59.212+0000] {taskinstance.py:2866} INFO - Starting attempt 14 of 14
[2025-01-30T04:37:59.237+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): copia> on 2025-01-30 03:32:35.404173+00:00
[2025-01-30T04:37:59.248+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1373) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-30T04:37:59.250+0000] {standard_task_runner.py:72} INFO - Started process 1375 to run task
[2025-01-30T04:37:59.250+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etapa2_dag', 'copia', 'manual__2025-01-30T03:32:35.404173+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/etapa2_dag.py', '--cfg-path', '/tmp/tmpxpcoct8v']
[2025-01-30T04:37:59.252+0000] {standard_task_runner.py:105} INFO - Job 54: Subtask copia
[2025-01-30T04:37:59.326+0000] {task_command.py:467} INFO - Running <TaskInstance: etapa2_dag.copia manual__2025-01-30T03:32:35.404173+00:00 [running]> on host eed2ce10479b
[2025-01-30T04:37:59.452+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etapa2_dag' AIRFLOW_CTX_TASK_ID='copia' AIRFLOW_CTX_EXECUTION_DATE='2025-01-30T03:32:35.404173+00:00' AIRFLOW_CTX_TRY_NUMBER='14' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-30T03:32:35.404173+00:00'
[2025-01-30T04:37:59.455+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-30T04:37:59.456+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-30T04:37:59.456+0000] {logging_mixin.py:190} INFO - Current task name:copia state:running start_date:2025-01-30 04:37:59.196506+00:00
[2025-01-30T04:37:59.457+0000] {logging_mixin.py:190} INFO - Dag name:etapa2_dag and current dag run status:running
[2025-01-30T04:37:59.457+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-30T04:37:59.506+0000] {logging_mixin.py:190} INFO - Table: categories -- parquetfile: /data/postgres/categories/2025-01-30/file.parquet
[2025-01-30T04:37:59.770+0000] {logging_mixin.py:190} INFO - Tabela categories já existe. Atualizando dados...
[2025-01-30T04:37:59.817+0000] {logging_mixin.py:190} INFO - Table: customers -- parquetfile: /data/postgres/customers/2025-01-30/file.parquet
[2025-01-30T04:37:59.840+0000] {logging_mixin.py:190} INFO - Tabela customers não existe. Criando a tabela...
[2025-01-30T04:37:59.856+0000] {logging_mixin.py:190} INFO - Erro ao atualizar ou criar tabela customers: column "none" does not exist
LINE 1: ...Sales Representative', 'Obere Str. 57', 'Berlin', None, '122...
                                                             ^
HINT:  Perhaps you meant to reference the column "customers.phone".
[2025-01-30T04:37:59.869+0000] {logging_mixin.py:190} INFO - Table: customer_customer_demo -- parquetfile: /data/postgres/customer_customer_demo/2025-01-30/file.parquet
[2025-01-30T04:37:59.887+0000] {logging_mixin.py:190} INFO - Tabela customer_customer_demo já existe. Atualizando dados...
[2025-01-30T04:37:59.912+0000] {logging_mixin.py:190} INFO - Table: customer_demographics -- parquetfile: /data/postgres/customer_demographics/2025-01-30/file.parquet
[2025-01-30T04:37:59.929+0000] {logging_mixin.py:190} INFO - Tabela customer_demographics já existe. Atualizando dados...
[2025-01-30T04:37:59.956+0000] {logging_mixin.py:190} INFO - Table: employees -- parquetfile: /data/postgres/employees/2025-01-30/file.parquet
[2025-01-30T04:37:59.977+0000] {logging_mixin.py:190} INFO - Tabela employees não existe. Criando a tabela...
[2025-01-30T04:37:59.995+0000] {logging_mixin.py:190} INFO - Erro ao atualizar ou criar tabela employees: column "nan" does not exist
LINE 1: ...rce, and the Pacific Rim Importers Association.', nan, 'http...
                                                             ^
[2025-01-30T04:38:00.009+0000] {logging_mixin.py:190} INFO - Table: employee_territories -- parquetfile: /data/postgres/employee_territories/2025-01-30/file.parquet
[2025-01-30T04:38:00.026+0000] {logging_mixin.py:190} INFO - Tabela employee_territories já existe. Atualizando dados...
[2025-01-30T04:38:00.167+0000] {logging_mixin.py:190} INFO - Table: orders -- parquetfile: /data/postgres/orders/2025-01-30/file.parquet
[2025-01-30T04:38:00.185+0000] {logging_mixin.py:190} INFO - Tabela orders não existe. Criando a tabela...
[2025-01-30T04:38:00.199+0000] {logging_mixin.py:190} INFO - Erro ao atualizar ou criar tabela orders: column "none" does not exist
LINE 1: ...cools Chevalier', '59 rue de l''Abbaye', 'Reims', None, '511...
                                                             ^
[2025-01-30T04:38:00.212+0000] {logging_mixin.py:190} INFO - Table: products -- parquetfile: /data/postgres/products/2025-01-30/file.parquet
[2025-01-30T04:38:00.228+0000] {logging_mixin.py:190} INFO - Tabela products não existe. Criando a tabela...
[2025-01-30T04:38:00.406+0000] {logging_mixin.py:190} INFO - Table: region -- parquetfile: /data/postgres/region/2025-01-30/file.parquet
[2025-01-30T04:38:00.421+0000] {logging_mixin.py:190} INFO - Tabela region já existe. Atualizando dados...
[2025-01-30T04:38:00.451+0000] {logging_mixin.py:190} INFO - Table: shippers -- parquetfile: /data/postgres/shippers/2025-01-30/file.parquet
[2025-01-30T04:38:00.466+0000] {logging_mixin.py:190} INFO - Tabela shippers já existe. Atualizando dados...
[2025-01-30T04:38:00.499+0000] {logging_mixin.py:190} INFO - Table: suppliers -- parquetfile: /data/postgres/suppliers/2025-01-30/file.parquet
[2025-01-30T04:38:00.516+0000] {logging_mixin.py:190} INFO - Tabela suppliers não existe. Criando a tabela...
[2025-01-30T04:38:00.529+0000] {logging_mixin.py:190} INFO - Erro ao atualizar ou criar tabela suppliers: column "none" does not exist
LINE 1: ...'Purchasing Manager', '49 Gilbert St.', 'London', None, 'EC1...
                                                             ^
HINT:  Perhaps you meant to reference the column "suppliers.phone".
[2025-01-30T04:38:00.540+0000] {logging_mixin.py:190} INFO - Table: territories -- parquetfile: /data/postgres/territories/2025-01-30/file.parquet
[2025-01-30T04:38:00.558+0000] {logging_mixin.py:190} INFO - Tabela territories já existe. Atualizando dados...
[2025-01-30T04:38:00.678+0000] {logging_mixin.py:190} INFO - Table: us_states -- parquetfile: /data/postgres/us_states/2025-01-30/file.parquet
[2025-01-30T04:38:00.693+0000] {logging_mixin.py:190} INFO - Tabela us_states já existe. Atualizando dados...
[2025-01-30T04:38:00.796+0000] {python.py:240} INFO - Done. Returned value was: True
[2025-01-30T04:38:00.829+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-30T04:38:00.829+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etapa2_dag, task_id=copia, run_id=manual__2025-01-30T03:32:35.404173+00:00, execution_date=20250130T033235, start_date=20250130T043759, end_date=20250130T043800
[2025-01-30T04:38:00.842+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-30T04:38:00.843+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-30T04:38:00.843+0000] {logging_mixin.py:190} INFO - Dag name:etapa2_dag queued_at:2025-01-30 04:37:51.208049+00:00
[2025-01-30T04:38:00.844+0000] {logging_mixin.py:190} INFO - Task hostname:eed2ce10479b operator:PythonOperator
[2025-01-30T04:38:00.871+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-30T04:38:00.895+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-30T04:38:00.899+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
